config/routes.rb

```diff
@@ -1,3 +1,12 @@
 Rails.application.routes.draw do
+  resources :posts
+  #resources :pages
+  get '/pages', to: 'pages#index'
+  post '/pages', to: 'pages#create'
+  get '/pages/new', to: 'pages#new', as: 'new_page'
+  get '/pages/:id', to: 'pages#show', as: 'page'
+  get '/pages/:id/edit', to: 'pages#edit', as: 'edit_page'
+  patch '/pages/:id', to: 'pages#update'
+  delete '/pages/:id', to: 'pages#destroy'
```

Comment:
It's good to know how to write out routes so this is completely fine. But I wanted to let you know that in the future you can use `resource`
"Resource routing allows you to quickly declare all of the common routes for a given resourceful controller. A single call to resources can declare all of the necessary routes for your index, show, new, edit, create, update, and destroy actions." - [Rails Resource Routing](https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default)